### ISSUE TITLE
xSQLServerSetup Features property is case sensitive 

### ISSUE THREAD

**Issue by KevinMarquette** [PowerShell/xSQLServer#38]
Installing with this value `Features = "SQLEngine"` produces this error message when applying the DSC configuration:

```
PowerShell DSC resource MSFT_xSQLServerSetup  failed to execute Set-TargetResource functionality with error message: Test-TargetResource
returned false after calling set.
    + CategoryInfo          : InvalidOperation: (:) [], CimException
    + FullyQualifiedErrorId : ProviderOperationExecutionFailure
```

The SQL install log provides this message:

```
Overall summary:
  Final result:                  Failed: see details below
  Exit code (Decimal):           -2061893627
  Exit facility code:            1306
  Exit error code:               5
  Exit message:                  Missing system administrator account. To continue, provide at least one Windows account to provision as a SQL Server system administrator.
  Start time:                    2016-03-07 15:22:41
  End time:                      2016-03-07 15:23:04
  Requested action:              Install
```

In the Set-TargetResource, the check to validate the list of features uses the array contains function `if($Features.Contains("SQLENGINE"))` and when it is used this way, it will not find `"SQLEngine"` in the collection.

**Comment by jpmnteiro** [comment]
I submitted PR #46 that solves this issue and adds some more logging to that particular area (painful to know why it's failing with the current generic message).

**Comment by johlju** [comment]
This issue has been resolved in merged PR #132
